src/modals/confirmation-modal/d-app-operations-confirmation/sign-payload-request-confirmation/sign-payload-request-confirmation.tsx

```diff
@@ -72,12 +81,16 @@ export const SignPayloadRequestConfirmation: FC<Props> = ({ message }) => {
         <AccountDropdownItem account={approver} />
         <Divider />
         <View style={styles.descriptionContainer}>
-          <Divider size={formatSize(12)} />
           <Text style={styles.descriptionText}>Payload to sign</Text>
-          <Divider size={formatSize(12)} />
+          <TextSegmentControl
+            width={formatSize(138)}
+            selectedIndex={payloadTypeIndex}
+            values={['Raw', 'Bytes']}
+            onChange={setPayloadTypeIndex}
+          />
```

Comment:
think about the case when parsing of payload message is going to be unsuccessful